src/pci.c

```diff
@@ -498,16 +529,25 @@ static int pci_program_bar(uint8_t bus, uint8_t dev, uint8_t fun,
     return ret;
 }
 
+#if defined(DEBUG_PCI)
 static void pci_dump_id(uint8_t bus, uint8_t dev, uint8_t fun)
 {
     uint16_t vid, did;
 
     vid = pci_config_read16(bus, dev, fun, PCI_VENDOR_ID_OFFSET);
     did = pci_config_read16(bus, dev, fun, PCI_DEVICE_ID_OFFSET);
 
-    PCI_DEBUG_PRINTF("PCI enum: dev: %x:%x.%x vid: %x did: %x\r\n",
+    PCI_DEBUG_PRINTF("\r\n\r\nPCI: %x:%x.%x %x:%x\r\n",
                     bus, dev, fun, (int)vid, (int)did);
 }
+#else
+static inline void pci_dump_id(uint8_t bus, uint8_t dev, uint8_t fun)
```

Comment:
this could be 
`#define pci_dump_id(...) do{}while(0)`  to save one symbol allocation